bootstrap.go

```diff
@@ -41,7 +41,12 @@ func Run(ec *config.Extended, opts ...Option) {
 		err error
 	)
 
-	bootstrapLogger := zap.New(zap.NewJSONEncoder())
+	bl, _ := zap.NewProduction()
```

Comment:
You really should check that error and panic or call core `log.Fatal(err)` on it; I'd suggest the latter:
```
import coreLog "log"

bl, err := zap.NewProduction()
if err != nil { coreLog.Fatal(err) }
```